docs/_sidebar.md

```diff
@@ -12,6 +12,7 @@
   - [Managing Nodes](usage/nodes-management.md)
   - [Accessing From Behind a Proxy](usage/reverse-proxy.md)
   - [Driver function](usage/driver_function.md)
+  - [Network Graph](usage/network_graph.md)
```

Comment:
it would be better if this
![image](https://user-images.githubusercontent.com/17641229/188591162-3199ab73-7e24-4963-948f-c2eac5ef1c23.png)
was visible at the top level.

Reply:
Ok no problem, thought there could be some more things to add to network graph feature